src/Core/src/Dispatching/DispatcherProvider.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.ComponentModel;
+
+namespace Microsoft.Maui.Dispatching
```

Comment:
I suggest changing the namespace from `Microsoft.Maui.Dispatching` -> `Microsoft.Maui.Dispatch`. It's where all the dispatch related code goes. And it's a bit shorter.